src/shared/jump_to_bottom_button.rs

```diff
@@ -178,3 +232,12 @@ impl JumpToBottomButtonRef {
         }
     }
 }
+
+/// The number of unread messages in the room
+#[derive(Clone, Debug)]
+pub enum UnReadMessageCount {
```

Comment:
be consistent with naming: "Unread" is one word.
```suggestion
pub enum UnreadMessageCount {
```